change-address/LETTERSPECS/BANNO.CHANGE.ADDR.V1.CFG

```diff
@@ -155,36 +173,36 @@ OE:
 *  This indicates whether a confirmation email will be sent to the member. SENDMAIL
 *  functionality must be turned on in your system
 *  VALID VALUES: YES or NO
-*  DEFAULT: NO (or left blank) 
-SE:
+*  DEFAULT: NO (or left blank)
+SE:YES
```

Comment:
This was previously blank so the default of NO was what was used. Is there a reason that it is now being set to YES?

Reply:
@jordan-keenan Corrected